cmd/scaffold.go

```diff
@@ -272,6 +272,27 @@ func (fnb *FlowNodeBuilder) EnqueuePingService() {
 	fnb.Component("ping service", func(node *NodeConfig) (module.ReadyDoneAware, error) {
 		pingLibP2PProtocolID := protocols.PingProtocolId(node.SporkID)
 
+		var hotstuffViewFunc func() (uint64, error)
+		// Setup consensus nodes to report their HotStuff view
+		if fnb.BaseConfig.NodeRole == flow.RoleConsensus.String() {
+			persist, err := persister.New(node.DB, node.RootChainID)
```

Comment:
Here we could then use  https://github.com/onflow/flow-go/blob/78e76d85b322d727dd6bf608cd1f6c9cca690a13/cmd/util/cmd/read-hotstuff/cmd/reader.go#L11-L20